Source/Plugin.BLE/Shared/ConnectParameter.cs

```diff
@@ -26,10 +32,14 @@ public struct ConnectParameters
         /// </summary>
         /// <param name="autoConnect">Android only: Whether to directly connect to the remote device (false) or to automatically connect as soon as the remote device becomes available (true). The default is false.</param>
         /// <param name="forceBleTransport">Android only: For Dual Mode device, force transport mode to LE. The default is false.</param>
-        public ConnectParameters(bool autoConnect = false, bool forceBleTransport = false)
+        public ConnectParameters(
+            bool autoConnect = false,
+            bool forceBleTransport = false,
+            ConnectionParameterSet connectionParameterSet = ConnectionParameterSet.None)
```

Comment:
Maybe add a brief documentation of the third param, analogous to the existing two? There are warnings about this in GHA.